### ISSUE TITLE
Perform (small) word embeddings on v1 synthetic data set

### ISSUE THREAD

**Issue by rockladyeagles** [vgcagle/DARC#11]
Write code that will, given a rewritten sentence from @rpersing (in whatever form you two decide) produce a NumPy array (or whatever you and @vgcagle decide) containing the GloVe (or word2vec) embeddings for each consecutive words. For the special "words" `__NE_FROM__` and `__NE_TO__`, you'll use a special/weird embedding as described in Wichmann et al.

"Small" means "as small as possible"; i.e., a few number of dimensions (to minimize processing lag at this development stage).

Open question: Should you use pre-trained embeddings, or train a model yourself to produce them? Advantages of the former include not having to figure out GloVe training code, and the fact that their pre-trained embeddings would be more comprehensive than training on our synthetic corpus. Advantages of the latter include that you can (presumably) control the dimension of the embeddings, so you can use (say) 30-dimensional embeddings instead of 300-dimensional (which will make everything run faster).

**Comment by rockladyeagles** [comment]
get labels to @vgcagle along with the embeddings
